.github/workflows/channels-ci.yml

```diff
@@ -19,6 +23,7 @@ jobs:
         with:
           node-version-file: ".nvmrc"
           cache: npm
+          cache-dependency-path: 'webapp/package-lock.json'
```

Comment:
This needs to be manually specified now since it's not at the root of the repository